commit c53c73b4a97b86349ac8480fa20f67b5756a7ed6 (refs/changes/05/3505/3)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2018-02-27T23:51:41+00:00 (1 year, 7 months ago)
    
    TRITON-180 Move "sdc" zone to new Triton origin image
    Reviewed by: Julien Gilli <julien.gilli@joyent.com>
    Approved by: Julien Gilli <julien.gilli@joyent.com>

diff --git a/targets.json.in b/targets.json.in
index f4ee194..65997e7 100644
--- a/targets.json.in
+++ b/targets.json.in
@@ -685,9 +685,10 @@ cat <<EOF
     "build_platform": "20151126T062538Z",
     "image_name": "sdc",
     "image_description": "SDC tools/ops zone",
-    "image_uuid": "fd2cc906-8938-11e3-beab-4359c665ac99",
+    "// image_uuid": "triton-origin-multiarch-15.4.1@1.0.1",
+    "image_uuid": "04a48d7d-6bb5-4e83-8c3b-e60a99e0f48f",
     "pkgsrc": [
-        "mtr-0.82"
+        "mtr-0.86nb3"
     ],
     "repos": [
       {"url": "git@github.com:joyent/sdc-sdc.git"}
